src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, ComponentProps } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import * as DropdownSelect from '@radix-ui/react-select';
+import ChevronDownIcon from '../../assets/chevronDown.svg?react';
+import ChevronUpIcon from '../../assets/chevronUp.svg?react';
```

Comment:
I would only use one svg and then use css to flip the icon for the different states.